libvirt/resource_libvirt_network_test.go

```diff
@@ -12,6 +13,10 @@ import (
 )
 
 func TestAccLibvirtNetwork_Addresses(t *testing.T) {
+	if os.Getenv("TF_LIBVIRT_DISABLE_PRIVILEGED_TESTS") != "" {
```

Comment:
I wonder if we could use test grouping somehow to avoid this repetition:

https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks

However, I am afraid that it would break running a single test from the command line.

Or perhaps just a helper function `skipIfNotPrivileged`?

Reply:
I am in favor for the helper function.

For grouping test, we might add a follow-up issue were can research this.

E.g there are 2 things we need to consider:
 - terraform has now a way to run parallel tests, so we can enable this
 - terraform has maybe another way to solve the grouping of tests problem

Reply:
Helper function added. PTALA.

Reply:
thx zenix!   I will look during next week.

Reply:
@zeenix  I could not find where the helper is defined. If you can point me a link it would be helpfull

Reply:
https://github.com/dmacvicar/terraform-provider-libvirt/pull/600/files?file-filters%5B%5D=.go&file-filters%5B%5D=No+extension#diff-668c1760b802d50dda6cbd25290d8c5e

Reply:
i-e in ` libvirt/helpers_test.go` file.

Reply:
I personally think that `skipIfNotPrivileged` is more clear then `skipIfPrivilegedDisabled`.

A part of that, looks good!

Reply:
I didn't go for the former because that sounds like it checks if running privileged or not, while it checks if privileged operations are disabled or not.

Reply:
Ok thx for clarification! we can stick with your  name and review it later on. At the end it will be just a matter of sedding things :grin